routes/students.js

```diff
@@ -12,6 +12,51 @@ router.get('', async (req, res) => {
   }
 });
 
+router.post('/create', async(req, res) => {
+  try {
+    const student = req.body;
+    console.log(student);
+    const newStudent = await pool.query(
+      'INSERT INTO student (first_name, last_name, contact, site_id, student_group) VALUES ($1, $2, $3, $4, $5) RETURNING student_id',
+      [student.first_name, student.last_name, student.contact, student.site_id, student.student_group],
+    );
+    res.json(newStudent.rows[0]);
+  } catch (err) {
+    console.error(err.message);
+  }
+});
+
+router.post('/update-scores/:id', async(req, res) => {
+  try {
+    const student = req.body;
+    console.log(student);
+    const studentScores = await pool.query(
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'studentScores' is assigned a value but never used.